### ISSUE TITLE
OC 8.0.2: Local Shares not handled correctly.

### ISSUE THREAD

**Issue by ravermeister** [owncloud/music#371]
Hi, I've added my music files to owncloud via a local share (see screenshot)
![Local Share](http://www.rimkus.it/devel/owncloud/local_share.png)

When I now set this folder in the personal Settings as the music root Folder :
![Music root](http://www.rimkus.it/devel/owncloud/music_root.png)

I can not access any Song in the Music app:
![Empty Music Page](http://www.rimkus.it/devel/owncloud/music_content.png)

even when I Browse through the folder via the Files app, and then access one mp3,
the music app is loaded, but this empty screen appears too.

What can I do? 
does the music app works with local shares?

**Comment by ravermeister** [comment]
### Steps to reproduce
1. Add a local Share where the mp3 files are in sub directories
2. set this folder as the music root in the personal settings
3. either browse via the file explorer to a mp3 file and try to play it 
4. or load the music app

### Expected behaviour
The music files below my music root folder is shown in the music app

### Actual behaviour
an  empty music app appears saying there are no media files

### Server configuration
**Operating system**: Ubuntu 14.04-lts

**Web server:** Apache 2.4 mpm-itk

**Database:** mysql 5.5

**PHP version:** php 5.5.9

**ownCloud version:** (see ownCloud admin page) 8.0.2

**Updated from an older ownCloud or fresh install:** yes 7.0.4 to 8.0 to 8.0.2

**List of activated apps:** 
Enabled:
 - activity
 - documents
 - external
 - files
 - files_external
 - files_gdocs_viewer
 - files_locking
 - files_mv
 - files_pdfviewer
 - files_sharing
 - files_texteditor
 - files_trashbin
 - files_versions
 - files_videoviewer
 - firstrunwizard
 - galleryplus
 - mozilla_sync
 - music
 - ocusagecharts
 - ownnote
 - search_lucene
 - templateeditor
 - updater
 - youtubedl


**The content of config/config.php:**

```
<?php
$CONFIG = array (
  'instanceid' => 'oc7b4ffa019d',
  'passwordsalt' => 'xxxxxxxxxxxxx',
  'trusted_domains' =>
  array (
    0 => 'echo390.server4you.de',
    1 => 'cloud.rimkus.it',
    2 => 'rimkus.it',
    3 => 'cloud.rimkus.tk',
    4 => 'rimkus.tk',
  ),
  'datadirectory' => '/home/ravermeister/shares/cloud.rimkus.it',
  'dbtype' => 'mysql',
  'version' => '8.0.2.0',
  'dbname' => 'cloud.rimkus.tk',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'xxxxxx',
  'dbpassword' => 'xxxxxx',
  'installed' => true,
  'forcessl' => false,
  'theme' => '',
  'maintenance' => false,
  'loglevel' => 2,
  'log_rotate_size' => 26214400,
  'mail_smtpmode' => 'smtp',
  'mail_smtpname' => 'info@rimkus.tk',
  'mail_smtppassword' => 'xxxxxxxx',
  'mail_smtpsecure' => 'tls',
  'mail_from_address' => 'info',
  'mail_domain' => 'xxxxxx',
  'mail_smtpauthtype' => 'LOGIN',
  'mail_smtpauth' => true,
  'mail_smtphost' => 'xxxxxx',
  'mail_smtpport' => '587',
  'enable_previews' => true,
  'preview_libreoffice_path' => '/usr/bin/libreoffice',
  'redis' =>
  array (
    'host' => 'localhost',
    'port' => 6379,
    'timeout' => 0,
  ),
  'secret' => 'xxxxxxxx',
  'check_for_working_htaccess' => true,
  'appstoreurl' => 'https://api.owncloud.com/v1',
  'apps_paths' =>
  array (
    0 =>
    array (
      'path' => '/home/ravermeister/cloud.rimkus.it/apps',
      'url' => '/apps',
      'writable' => true,
    ),
  ),
  'appcodechecker' => false,
  'preview_max_scale_factor' => 1,
  'appcodechecker' => false,
  'preview_max_scale_factor' => 1,
  'enabledPreviewProviders' =>
  array (
    0 => 'OC\\Preview\\Image',
    1 => 'OC\\Preview\\Illustrator',
    2 => 'OC\\Preview\\Postscript',
    3 => 'OC\\Preview\\Photoshop',
    4 => 'OC\\Preview\\TIFF',
    5 => 'OC\\Preview\\Movies',
    6 => 'OC\\Preview\\MSOffice2003',
    7 => 'OC\\Preview\\MSOffice2007',
    8 => 'OC\\Preview\\MSOfficeDoc',
    9 => 'OC\\Preview\\OpenDocument',
    10 => 'OC\\Preview\\PDF',
    11 => 'OC\\Preview\\SVG',
  ),
  'filesystem_check_changes' => 1,
  'asset-pipeline.enabled' => true,
);
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
yes local

**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:** firefox

**Operating system:** Win 7 64

### Logs
#### Web server error log
```
37.201.240.153 - - [21/Apr/2015:18:12:29 +0200] "GET /index.php/apps/music/ HTTP/1.1" 200 4277 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:29 +0200] "GET /assets/bd078f01fcfa672f35285827c4759d06.css HTTP/1.1" 200 20295 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:29 +0200] "GET /index.php/core/js/oc.js?v=488d4c35112a2d247c18ff0451debb31 HTTP/1.1" 200 2650 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:29 +0200] "GET /assets/0173c2e15cef1a4c53671c6cbe657f03.js HTTP/1.1" 200 475919 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/img/actions/caret.svg HTTP/1.1" 200 2416 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /settings/img/personal.svg HTTP/1.1" 200 2010 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /settings/img/users.svg HTTP/1.1" 200 3344 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /settings/img/help.svg HTTP/1.1" 200 1258 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/img/actions/logout.svg HTTP/1.1" 200 2755 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /settings/img/admin.svg HTTP/1.1" 200 883 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/activity/img/activity.svg HTTP/1.1" 200 786 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/img/places/files.svg HTTP/1.1" 200 1112 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/documents/img/documents.svg HTTP/1.1" 200 1117 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/galleryplus/img/app.svg HTTP/1.1" 200 985 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/music/img/music.svg HTTP/1.1" 200 1106 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/youtubedl/img/app.svg HTTP/1.1" 200 5902 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/ownnote/img/app.svg HTTP/1.1" 200 1939 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/ocusagecharts/img/iconchart.png HTTP/1.1" 200 488 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/external/img/external.svg HTTP/1.1" 200 2402 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /settings/img/apps.svg HTTP/1.1" 200 631 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/music/img/play-previous.svg HTTP/1.1" 200 736 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/music/img/play-big.svg HTTP/1.1" 200 704 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/music/img/pause-big.svg HTTP/1.1" 200 717 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/music/img/play-next.svg HTTP/1.1" 200 735 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/music/img/shuffle.svg HTTP/1.1" 200 1715 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /apps/music/img/repeat.svg HTTP/1.1" 200 1175 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /index.php/avatar/ravermeister/32?requesttoken=ba4Pfr.LA3KX/.hL5KuDE1Nhyyph47 HTTP/1.1" 200 1946 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/img/logo-icon.svg HTTP/1.1" 200 3181 "http://cloud.rimkus.it/assets/bd078f01fcfa672f35285827c4759d06.css" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/img/actions/search.svg HTTP/1.1" 200 1115 "http://cloud.rimkus.it/assets/bd078f01fcfa672f35285827c4759d06.css" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/fonts/OpenSans-Regular.woff HTTP/1.1" 200 20844 "http://cloud.rimkus.it/assets/bd078f01fcfa672f35285827c4759d06.css" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/img/breadcrumb.svg HTTP/1.1" 200 742 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/search/templates/part.results.html HTTP/1.1" 200 484 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /core/img/loading.gif HTTP/1.1" 200 3492 "http://cloud.rimkus.it/assets/bd078f01fcfa672f35285827c4759d06.css" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /index.php/avatar/ravermeister/128?requesttoken=ba4Pfr.LA3KX/.hL5KuDE1Nhyyph47 HTTP/1.1" 200 6103 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /index.php/apps/music/api/collection HTTP/1.1" 200 648 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
37.201.240.153 - - [21/Apr/2015:18:12:30 +0200] "GET /index.php/apps/music/api/scan?dry=1 HTTP/1.1" 200 686 "http://cloud.rimkus.it/index.php/apps/music/" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:37.0) Gecko/20100101 Firefox/37.0"
```

#### ownCloud log (data/owncloud.log)
```
empty
```

**Comment by ravermeister** [comment]
..hope this is enough information to start to investigate....

**Comment by mossholderm** [comment]
This is hitting me as well with 8.1.1 .

**Comment by paulijar** [comment]
Local shares were fixed to work properly as part of PRs #564 and #566.
